src/IndexingWrapper.cpp

```diff
@@ -243,6 +245,71 @@ indexWrapper(Rcpp::CharacterVector & dbpath, Rcpp::DataFrame & dataFrame, Rcpp::
 	    }
 	}
 
+	//Add value slots
+	for (Rcpp::List::iterator it = slotList.begin(); it != slotList.end(); ++it) {
+	    Rcpp::List list = it.operator*();
```

Comment:
Why not just `*it`? (This is a common C++ pattern, so not using it should be called out with a comment and an explanation — but I think in this situation there's no good reason?)